net-nntp/sabnzbd/sabnzbd-4.1.0.ebuild

```diff
@@ -0,0 +1,168 @@
+		test? ( $(python_gen_cond_dep '
+				dev-python/tavalidate[${PYTHON_USEDEP}]
+				>=dev-python/tavern-2[${PYTHON_USEDEP}]
+			')
+		)
+"
+RDEPEND="
+	${DEPEND}
+	|| (
+		>=app-arch/par2cmdline-0.8
+		>=app-arch/par2cmdline-turbo-1.1.0
+	)
+	net-misc/wget
+"
+BDEPEND="
+	test? (
+		$(python_gen_cond_dep '
+			dev-python/flaky[${PYTHON_USEDEP}]
+			>=dev-python/lxml-4.5.0[${PYTHON_USEDEP}]
+			dev-python/pyfakefs[${PYTHON_USEDEP}]
+			dev-python/pytest-httpbin[${PYTHON_USEDEP}]
+			dev-python/pytest-httpserver[${PYTHON_USEDEP}]
+			dev-python/pytest[${PYTHON_USEDEP}]
+			dev-python/requests[${PYTHON_USEDEP}]
+			dev-python/selenium[${PYTHON_USEDEP}]
+			dev-python/werkzeug[${PYTHON_USEDEP}]
+			dev-python/xmltodict[${PYTHON_USEDEP}]
+		')
+		app-arch/p7zip
+		app-arch/unrar
+		app-arch/unzip
+		www-apps/chromedriver-bin
+	)
+"
+
+src_test() {
+	local EPYTEST_IGNORE=(
+		# network sandbox
+		tests/test_getipaddress.py
+		tests/test_rss.py
+		tests/test_urlgrabber.py
+		tests/test_utils/test_happyeyeballs.py
+		tests/test_utils/test_internetspeed.py
+	)
+	local EPYTEST_DESELECT=(
+		# network sandbox
+		'tests/test_cfg.py::TestValidators::test_validate_host'
+		'tests/test_consistency.py::TestWiki'
+		'tests/test_newswrapper.py::TestNewsWrapper'
+		# Doesn't work, complains mocker missing even when pytest-mock installed
+		'tests/test_dirscanner.py::TestDirScanner'
+		# Just plain fails
+		'tests/test_newsunpack.py::TestPar2Repair::test_basic'
+		# Chromedriver tests don't want to behave in portage
+		'tests/test_functional_config.py::TestBasicPages::test_base_pages'
+		'tests/test_functional_config.py::TestBasicPages::test_base_submit_pages'
+		'tests/test_functional_config.py::TestConfigLogin::test_login'
+		'tests/test_functional_config.py::TestConfigCategories::test_page'
+		'tests/test_functional_config.py::TestConfigRSS::test_rss_basic_flow'
+		'tests/test_functional_config.py::TestConfigServers::test_add_and_remove_server'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_basic_rar5'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_zip'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_7zip'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_passworded'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_fully_obfuscated'
+		'tests/test_functional_downloads.py::TestDownloadFlow::test_download_unicode_rar'
+		'tests/test_functional_misc.py::TestShowLogging::test_showlog'
+		'tests/test_functional_misc.py::TestQueueRepair::test_queue_repair'
+		'tests/test_functional_misc.py::TestDaemonizing::test_daemonizing'
+		'tests/test_functional_sorting.py::TestDownloadSorting'
+	)
+
+	local -x PYTEST_DISABLE_PLUGIN_AUTOLOAD=1
+	local -x PYTEST_PLUGINS=tavern._core.pytest
+	epytest -s
+}
+
+src_install() {
+	local d
+	for d in email icons interfaces locale po sabnzbd scripts tools; do
+		insinto /usr/share/${PN}/${d}
+		doins -r ${d}/*
+	done
```

Comment:
Any reason not to do the equivalent of:

```
insinto /usr/share/${PN}
doins -r email icons ...
```

?

Reply:
Didn't know I could, inherited from old EAPI where maybe was not possible. It's changed now, thanks!